internal/resources/resource_host.go

```diff
@@ -460,12 +485,17 @@ func resourceMetalHostRead(d *schema.ResourceData, meta interface{}) (err error)
 			vTargetIQN:   i.TargetIQN,
 		}
 		volumeInfos = append(volumeInfos, vi)
+		discoveryIPs = append(discoveryIPs, i.DiscoveryIP)
```

Comment:
could instead only append if unique.  would need to add to utils.go something like hpe-hcss/quake/pkg/util/slice_utils.go (AppendStrUnique).   I miss Java Sets...

Reply:
No need for the function anymore